src/tags/control/Taxonomy.js

```diff
@@ -83,7 +83,7 @@ function uniq(nodes) {
 function traverse(root) {
   const visitNode = function(node, parents = []) {
     const label = node.value;
-    const path = [...parents, label]; // @todo node.alias || label; problems with showFullPath
+    const path = [...parents, node.alias ?? node.value]; // @todo node.alias || label; problems with showFullPath
```

Comment:
this comment can be removed now that it's implemented?
```suggestion
    const path = [...parents, node.alias ?? label];
```